src/redis.nim

```diff
@@ -778,8 +777,7 @@ proc lLPush*(r: Redis | AsyncRedis, key: string, values: seq[string], create: bo
   result = await r.readInteger()
 
 proc lRange*(r: Redis | AsyncRedis, key: string, start, stop: int): Future[RedisList] {.multisync.} =
-  ## Get a range of elements from a list. Returns `nil` when `key`
-  ## doesn't exist.
```

Comment:
A Future can still be nil, no?

Reply:
Yes, but if async gets a nil future it will raise IIRC